scripts/cnv_wdl/germline/cnv_germline_case_workflow.wdl

```diff
@@ -242,6 +250,7 @@ workflow CNVGermlineCaseWorkflow {
         Array[File] gcnv_tracking_tars = GermlineCNVCallerCaseMode.gcnv_tracking_tar
         Array[File] genotyped_intervals_vcf = PostprocessGermlineCNVCalls.genotyped_intervals_vcf
         Array[File] genotyped_segments_vcf = PostprocessGermlineCNVCalls.genotyped_segments_vcf
+        Array[File] qc_status_files = CollectSampleQualityMetrics.qc_status_files
```

Comment:
Ideally I'd want to be able to flag failing samples in an obvious way in the workspace, like having new fields in the data model called "sample_quality" and "model_quality" with the QC status reported there.  Are we violently opposed to having a Cromwell version and a Firecloud version of this WDL?  (@LeeTL1220)

Reply:
I've done what Lee recommended, however note that I had to make the task process one sample at a time